### PR TITLE
New development project 2

### PR DESCRIPTION
**Pull request by monicah79**
I did button styling
changed colors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,8 +152,8 @@ <h2>About Myself</h2>
     
     <p class="homepage-paragraph">
         Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.    </p>
-    <p class="homepage-quotes">
-        LET'S CONNECT</p>
+    <h2 class="homepage-quotes">
+        LET'S CONNECT</h2>
```
Comment (franclobo) [inline index.html:156]: - [ ] Please 🙏🏽 Correct the `font-size` of "About my self" and the color of "LET'S CONNECT" to look like the Figmr template.

| **Figma template** | **Your design**|
|---|---|
| image | image
 |

File: index.html [hunk 2]
```diff
@@ -178,26 +178,23 @@ <h2>About Myself</h2>
     </section>
     <section class="sub-language">
         <div class="sub-column">
-            <div class="language-heading">
-				<p>Language</p>
-				<img src="Images/arrow-down-icon/Normal Button/Tertiary/Icons/Union.svg" alt=""/>
-			</div>
-             <div>
+            <li class="language-heading"> Language
```
Comment (franclobo) [inline index.html:181]: - [ ] Please 🙏🏽 correct the word "Language", it is plural "Languages" to be similar to the Figma template.

File: index.html [hunk 3]
```diff
@@ -178,26 +178,23 @@ <h2>About Myself</h2>
...
                 <ul class="mylist">
                     <li class="mylist-item"><img src="Images/javascript.svg" alt=""/> <span>Javascript</span> </li>
                     <li class="mylist-item"><img src="Images/html.svg" alt=""/> <span>Html</span> </li>
                     <li class="mylist-item"><img src="Images/css.svg" alt=""/> <span>Css</span> </li>
                       </ul>
              </div>
-        </div>
-        <div class="language-heading">
-            <p>Framework</p>
+        <li class="language-heading">Framework
```
Comment (franclobo) [inline index.html:190]: - [ ] Please 🙏🏽 correct the word "Framework", it is plural "Frameworks" to be similar to the Figma template.

File: style.css [hunk 4]
```diff
@@ -215,16 +252,30 @@ div.sub-row {
...
+  width: 327px;
   margin-top: 20px;
+  list-style-type: none;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  align-items: center;
+  color: #000;
+}
```
Comment (franclobo) [inline style.css:275]: - [ ] Kindly correct the margin of this section to look similar to the Figma template. 👇🏽 

| **Figma template** | **Your template** |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @monicah79, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please 🙏🏽 do not merge or delete a  branch that hasn't been approved before. You have deleted this [branch](https://github.com/monicah79/Personal-portfolio/pull/2) 👇🏽 
image

- [ ] Kindly use two lines for "Multi-Post Stories" and correct the color of "Back End dev 2015" in all the cards, it is grey and there are circles between the words, to look like the Figma template 👇🏽 

| **Figma template** | **Your design** |
|---|---|
| image | image |

- [ ] As the last code reviewer said 👉🏽 You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please make sure that your work section uses the font and colors (background and font) defined for the template and also **section has the margins (top, bottom, left, right) defined in the design.**

- **Your design**

image

image

- **Figma template**

image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by monicah79** [comment]
done the changes

**Pull request closed without merging**